Overview.src.html

```diff
@@ -4118,10 +4155,11 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  if <var>input</var> is a <code>Request</code> object, and null otherwise.
 
  <li><p>If either <var>init</var>'s <code title>body</code> member is present and is non-null or
- <var>inputBody</var> is non-null, and <var>request</var>'s
+ <var>inputBody</var> is non-null, then <span data-anolis-spec=webidl>throw</span> a
+ <code title>TypeError</code> if either <var>request</var>'s
  <span title=concept-request-method>method</span> is `<code title>GET</code>` or
- `<code title>HEAD</code>`, <span data-anolis-spec=webidl>throw</span> a
- <code title>TypeError</code>.
+ `<code title>HEAD</code>`, or <var>request</var>'s
+ <span title=concept-request-attached-credential>attached credential</span> is non-null.
```

Comment:
Maybe 
```
<p>If all of the following are true
<ul class=brief>
 <li><p>init's body member is present and is non-nonull or inputBody is non-null
 <li><p>request's method is GET or HEAD, or request's attached credential is non-null
</ul>
<p>then throw a ... exception.</p>
```
Basically use `<ul class=brief>` as something that is part of the paragraph even though it's not. Started using that here and there and I think it makes these kind of statements easier to read.